STACK-UP-3/warriorz-backend
# #171784081 Setup API documentation using swagger

#### What does this PR do?
Having swagger for API documentation.

#### Description of Task to be completed?
  Configure swagger to be used as API documentation engine
  Add babel configuration

#### How should this be manually tested?
After cloning the repo, cd into it, type npm install, then type npm start,
open the browser and go to http://localhost:5000/api-docs/json or http://localhost:5000/api-docs
you see the swagger page in JSON for case 1 and swagger-ui in case 2

#### What are the relevant pivotal tracker stories?
[#171784081](https://www.pivotaltracker.com/n/projects/2438874/stories/171784081)
